### DISCUSSION TITLE
Husky needed

### DISCUSSION THREAD

**Discussion by alnavarrop99** [novajslabs/nova.js#77] [Q&A]
It would be very useful to install Husky for the repo and now prevent commits with errors locally. In the future, it would be a good idea to create a GitHub workflow that also verifies all this.

Recommendation: 
- Husky Tools [lint-staged](https://github.com/lint-staged/lint-staged#readme)
- Linter: [oxlint](https://oxc-project.github.io/)

**Comment by alnavarrop99** [comment]
@dlcastillop

**Reply by dlcastillop** [reply]
Sorry for the delay in answering, I'm very busy lately. It seems like a good idea to me, I just have to see the tools you mention since I have never used them before. I'll open an issue about it. Thank you so much. 🙌
